doc/source/server_architecture.rst

```diff
@@ -0,0 +1,86 @@
+AIT Server Introduction
+========================
+
+The core of AIT is a "server", :mod:`ait.core.server`, which manages the processing of telemetry, logging, and command messages. The server manages relationships between the following components: 
+
+* **inbound streams**   - inbound streams manage incoming data being recieved by AIT, for example telemtry messages coming into a port.
+* **outbound streams**   - outbound streams manage outgoing data being published by AIT, for example command messages being published to a port.
+* **handlers**   - handlers do the processing of data for inbound and outbound streams. When a message is received on an inbound stream for instance, a handler can handle the processing that needs to be done on the message before it is passed off to outbound streams or plugins.
+* **plugins**       - plugins provide functionality that consume data from inbound streams, do something with it, and provide data to outbound streams. The AIT GUI is a plugin.
+
+The AIT Server uses the messaging library ZeroMQ to manage the passing of data between streams and plugins.
+
+Streams, Plugins and Handlers
+-----------------------------
+
+Data will generally flow through the components as follows:
+
+-> Inbound Stream -> (Handlers) -> Plugin -> Outbound Stream -> (Handlers) ->
+
+* Plugins can have any number of inbound streams as inputs and any number of outbound streams as outputs. 
+* Inbound streams can have either integer ports or other inbound stream names as inputs.
+* Outbound streams can have either plugin or inbound stream names as inputs. They can output to other outbound streams or to an integer port.
```

Comment:
I think this first sentence should say **outbound** stream not inbound

Reply:
Outbound streams can have either inbound or outbound streams as inputs. A plugin is not required.